HopsanGUI/PlotTab.cpp

```diff
@@ -1468,7 +1533,47 @@ void PlotTab::openCreateBodePlotDialog()
         const double dataSize = pTimeVector->getDataSize()+1;
         const double stopTime = pTimeVector->last();
         const double maxFreq = dataSize/stopTime/2;
-        QLabel *pMaxFrequencyLabel = new QLabel("Maximum frequency in bode plot:");
+
+        QGroupBox *pWindowingGroupBox = new QGroupBox("Windowing", pBodeDialog);
+        QGridLayout *pWindowingLayout = new QGridLayout(pWindowingGroupBox);
+
+        QLabel *pWindowingLabel = new QLabel("Windowing function:",pBodeDialog);
+        QComboBox *pWindowingComboBox = new QComboBox(pBodeDialog);
+        pWindowingComboBox->addItems(QStringList() << "Rectangular" << "Hann Function");
+        pWindowingComboBox->setCurrentIndex(1);
```

Comment:
Default should be rectangular like in hcom, and I realized its the same as do nothing